scripts/k3d/smoke.sh

```
@@ -7,18 +7,21 @@ if (( BASH_VERSINFO[0] > 4 || ( BASH_VERSINFO[0] == 4 && BASH_VERSINFO[1] >= 4 )
 fi
 
 ROOT="$(git rev-parse --show-toplevel)"
+# shellcheck source=scripts/k3d/_load-dotenv.sh
+source "${ROOT}/scripts/k3d/_load-dotenv.sh"
```

Shellcheck (0.11.0)

[info] Not following: scripts/k3d/_load-dotenv.sh was not specified as input (see shellcheck -x).

(SC1091)